ddl/table.go

```diff
@@ -72,35 +72,21 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 		return ver, errors.Trace(ErrPlacementPolicyWithDirectOption.GenWithStackByArgs(tbInfo.PlacementPolicyRef.Name))
 	}
 	var bundle *placement.Bundle
-	if tbInfo.DirectPlacementOpts != nil {
-		// check the direct placement option compatibility.
-		if err := checkPolicyValidation(tbInfo.DirectPlacementOpts); err != nil {
-			return ver, errors.Trace(err)
-		}
-		// build bundle from direct placement options.
-		bundle, err = placement.NewBundleFromOptions(tbInfo.DirectPlacementOpts)
-		if err != nil {
-			return ver, errors.Trace(err)
-		}
-	}
-	if tbInfo.PlacementPolicyRef != nil {
-		// placement policy reference will override the direct placement options.
-		po, err := checkPlacementPolicyExistAndCancelNonExistJob(t, job, tbInfo.PlacementPolicyRef.ID)
-		if err != nil {
-			return ver, errors.Trace(infoschema.ErrPlacementPolicyNotExists.GenWithStackByArgs(tbInfo.PlacementPolicyRef.Name))
-		}
-		// build bundle from placement policy.
-		bundle, err = placement.NewBundleFromOptions(po.PlacementSettings)
-		if err != nil {
-			return ver, errors.Trace(err)
-		}
+	dbInfo, err := checkSchemaExistAndCancelNotExistJob(t, job)
```

Comment:
I think checking exist of schema is not necessary because we already did it in `checkTableNotExists` at line 63

Reply:
just wanna get the dbInfo